examples/job_attachments/manifest_snapshot.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
I noticed you included output in only the first bash example, but not the following examples. Any reasons why, could we be consistent? I found having at least the schema is helpful.

Reply:
Yeah, it was something I was thinking about. I'll add them :)